### PR TITLE
Tasks page

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Hey folks, This is my PR for Tasks page. Please click below link to check specific files

[Click here to check](https://github.com/bhushannaikaroskar/pomodoro-app/pull/1/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&show-viewed-files=true)

### CODE REVIEW

File: src/components/task-item/TaskItem.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./taskitem.css"
+
+export default function TaskItem(task) {
+    const {  title } = task;
+
+    return (
+        <div className="task-item" onClick={()=>{console.log("clicked")}}>
```
Comment (archit7448) [inline src/components/task-item/TaskItem.jsx:8]: you should not console log click and If it is button you can show it in button tag.

**Review by archit7448 — APPROVED** [review]
- code looks structured
- you can add images or netlify link for PR review
- looks good to merge

File: src/components/task-item/TaskItem.jsx (hunk 1 above)
Comment (bhushannaikaroskar) [reply src/components/task-item/TaskItem.jsx:8]: actually wanted to make the TaskItem as whole clickable and it contains icons as well so I resorted to this approach.

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by sudipkundu999 — APPROVED** [review]
The code looks good. Good Work

**Pull request merged**